common/v2/features/SendAssets/components/SendAssetsForm.tsx

```diff
@@ -6,16 +6,18 @@ import { Button, Input } from '@mycrypto/ui';
 import { WhenQueryExists } from 'components/renderCbs';
 import { DeepPartial } from 'shared/types/util';
 import translate, { translateRaw } from 'translations';
-import { fetchGasPriceEstimates, getNonce } from 'v2';
-import { AccountContext } from 'v2/providers';
+import { getNonce } from 'v2/services/EthService';
+import { fetchGasPriceEstimates } from 'v2/services/ApiService';
+import { AccountContext, getAssetByUUID, getNetworkByName } from 'v2/services/Store';
 import {
-  ExtendedAccount as IExtendedAccount,
-  AssetBalanceObject,
   Asset,
-  ExtendedAccount
-} from 'v2/services';
-// import { processFormDataToTx } from 'v2/libs/transaction/process';
-import { IAsset, TSymbol } from 'v2/types';
+  IAsset,
+  TSymbol,
+  AssetBalanceObject,
+  ExtendedAccount as IExtendedAccount
```

Comment:
Why is it imported as `IExtendedAccount` here?

Reply:
A believe it's just legacy and didn't want to change the name.
I think we should get rid of the id of `Extended<DataType>` which just ads an uuid to the type and instead have all <DataType> inherit from an object that expects an uuid.